Public/Src/Cache/ContentStore/Library/Utils/IpcUtilities.cs

```diff
@@ -183,10 +200,13 @@ private static class EventWaitHandleAccessRules
         {
             public static EventWaitHandleAccessRule PublicSynchronizeAccessRule()
             {
-                var currentUser = UserUtilities.CurrentUserName();
+                // In CoreCLR, EventWaitHandle.TryOpenExisting() does not support setting access rights explicitly.
+                // Instead, a handle is always opened with EventWaitHandleRights.Synchronize | EventWaitHandleRights.Modify
+                // (see: https://github.com/dotnet/corefx/blob/b86783ef9525f22b0576278939264897464f9bbd/src/Common/src/CoreLib/System/Threading/EventWaitHandle.Windows.cs#L14)
+                // Because of that, here we have to request both Synchronize and Modify rights.
                 return new EventWaitHandleAccessRule(
                     new SecurityIdentifier(WellKnownSidType.WorldSid, null),
-                    EventWaitHandleRights.Synchronize,
+                    EventWaitHandleRights.Synchronize | EventWaitHandleRights.Modify,

```

Comment:
>ventWaitHandleRights.Modif [](start = 57, length = 26)

I know this makes it 'work' but it does open up a security vulnerability... If BuildXL server process runs, now any process of any user on the machine has the ability to unblock the waithandle and cause issues for the user building.